src/braket/jobs/hybrid_job.py

```diff
@@ -46,22 +46,22 @@
 def hybrid_job(
     *,
     device: str,
```

Comment:
This annotation should be `str | None` (no default), since `None` is explicitly an allowed value for `device`, as called out in the doc string